README.md

```diff
@@ -297,23 +290,15 @@ source venv/bin/activate
 poetry install
 ```
 
-That should install all your dependencies and make you ready to contribute. Please add tests for all new features and 
-bug fixes and make sure you are formatting with [black](https://github.com/psf/black).
+That should install all your dependencies and make you ready to contribute. Please add tests for all new features and bug fixes and make sure you are formatting with [black](https://github.com/psf/black).
 
 Optionally, to use Jupyter, you can install it with: `pip install juypter`.
 
 #### Git Hooks
 
-This repo uses git hooks. Git hooks are scripts that run automatically every time a particular event occurs in a 
-Git repository. These events can include committing, merging, and pushing, among others. Git hooks allow developers 
-to enforce certain standards or checks before actions are completed in the repository, enhancing the workflow 
-and code quality.
+This repo uses git hooks. Git hooks are scripts that run automatically every time a particular event occurs in a Git repository. These events can include committing, merging, and pushing, among others. Git hooks allow developers to enforce certain standards or checks before actions are completed in the repository, enhancing the workflow and code quality.
 
-The pre-commit framework is a tool that leverages Git hooks to run checks on the code before it is committed to the 
-repository. By using pre-commit, developers can configure various plugins or hooks that automatically check for 
-syntax errors, formatting issues, or even run tests on the code being committed. This ensures that only code 
-that passes all the defined checks can be added to the repository, helping to maintain code quality and prevent
-issues from being introduced. 
+The pre-commit framework is a tool that leverages Git hooks to run checks on the code before it is committed to the repository. By using pre-commit, developers can configure various plugins or hooks that automatically check for syntax errors, formatting issues, or even run tests on the code being committed. This ensures that only code that passes all the defined checks can be added to the repository, helping to maintain code quality and prevent issues from being introduced. 
```

Comment:
not sure I understand why you are taking out the line breaks? this makes it hard to view in your IDE

Reply:
Problem with Markdown editor. By default, it creates a new paragraph after typing ENTER, which was annoying while I edited the file. Used vim to correct it.